catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/security/WorkspacePolicyExtension.java

```diff
@@ -127,15 +128,29 @@ public KeyValueCollectionPermission isPermittedMatchAll(
                   .filter((permission) -> !implied.contains(permission.getKey()))
                   .collect(Collectors.toList());
 
-          return new KeyValueCollectionPermission(match.getAction(), values);
+          if (values.isEmpty()) {
```

Comment:
Do we still want this if else @djblue

Reply:
Since we showed that both are equivalent, I would prefer the original.

Reply:
Reverted.